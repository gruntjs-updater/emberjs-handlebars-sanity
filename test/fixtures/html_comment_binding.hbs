Some content <!-- 
that will be 
<strong>{{bound}}</strong>
incorrectly
-->
but looks normal