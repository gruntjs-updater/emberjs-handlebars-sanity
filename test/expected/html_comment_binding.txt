Found binding 'bound' within HTML comment
